src/utils/rootkeypackage_utils/src/rootkeypackage_do_download.cpp

```diff
@@ -13,6 +13,8 @@
 
 EXTERN_C_BEGIN
 
+#define MAX_TIME_TO_DOWNLOAD_IN_HOURS 1 /* hour*/
```

Comment:
I mentioned this previously and can't find my comment?  See the mail I sent on std::chrono.  C++ code shouldn't have `1 /* hour */`

Reply:
tl;dr -- make this a strongly typed const with `= std::chrono::hours(1)`